.gitignore

```diff
@@ -10,6 +10,7 @@ CMakeCache.txt
 nodes/
 .ipynb_checkpoints/
 /docker-images/
+fetch-ledger-develop-image
```

Comment:
generally speaking this is non-ideal. presumably this is the name of your symbolic link to the docker-images repo - but not everyone necessarily names it the same thing, so this wouldn't work for others anyway.

What you really want to do us edit your excludes file:
.git/info/exclude

it does exactly the same thing as gitignore but will only affect your local git repo and wont get pushed to remote